eslint.config.js

```diff
@@ -37,7 +37,8 @@ export default [
       '**/*.d.ts',
       '**/dist/**/*',
       '**/coverage/**/*',
-      'packages/**/client/**/*'
+      'packages/**/client/**/*',
+      'packages/prompt-registry/src/zod/*'
```

Comment:
[q] I see that after zod generation, `pnpm lint:fix` is executed. Should we then allow checking for those generated files as they are going to be part of our codebase?

Reply:
I was in two minds about it. I'll add linting for them, but ignore some of the rules like jsdoc/require-documentation.